### ISSUE TITLE
request.continue() in intercept does not update POST parameters 

### ISSUE THREAD

**Issue by rmallof** [getgauge/taiko#1520]
### Expected behavior
The code below, using intercept and request.continue(), should intercept and overwrite POST request parameters

### Actual behavior
The code below successfully captures the POST request, but the modification has no effect - i.e., data reaching the server is the original one, not the overwritten by intercept

### Steps to reproduce
1. Use the intercept functionality with the form at https://tryphp.w3schools.com/showphp.php?filename=demo_form_post while intercepting https://tryphp.w3schools.com/demo/welcome.php. Code example:

```
step(["Step"], async () => {
    await intercept("https://tryphp.w3schools.com/demo/welcome.php", (request) => { 
        console.log("@ Intercept step callback");
        let parameters = querystring.parse(request.request.postData)
        parameters["name"] = "This comes from Taiko";    <--------------- overwritting "name" variable with "This comes from Taiko"
        request.request.postData = querystring.stringify(parameters);
        request.continue({ postData: request.request.postData });
    });
});

```
2. Run the above code with gauge + taiko
3. The data shown at https://tryphp.w3schools.com/showphp.php?filename=demo_form_post after sending the form is the one written in the textBox instead of "This comes from Taiko"

FYI, chatting about this very issue @ https://github.com/getgauge/taiko/issues/98#issuecomment-692928634

### Versions
```
gauge --version
Gauge version: 1.1.1
Commit Hash: 6e0d83f

taiko --version
Version: 1.0.22 (Chromium: 86.0.4240.0) RELEASE
```

Thanks

**Comment by NivedhaSenthil** [comment]
Fixed via #1525

**Comment by rmallof** [comment]
@NivedhaSenthil good job, it's working now :) 
Really appreciated!
